apps/api/src/auth/routes/api-keys/api-keys.router.ts

```diff
@@ -1,9 +1,15 @@
+import {
+  CreateApiKeyRequestSchema,
+  FindApiKeyParamsSchema,
+  ListApiKeysResponseSchema,
+  SingleApiKeyResponseSchema,
+  UpdateApiKeyRequestSchema
+} from "@akashnetwork/http-sdk";
```

Comment:
I'm not sure I follow why these are in http package and not in the app close to the router. Basically API is the source of those. Http SDKs are based on the API and should utilise what it has to export.

blocking at least till we discus this.

Reply:
I put the types back in the API and left the types needed for the http-sdk